ckanext/fluent/validators.py

```diff
@@ -23,6 +23,23 @@ def noop(key, data, errors, context):
         return noop
     validators_from_string = None
 
+
+def fluent_core_field_output(key, data, errors, context):
+    """
+    Return a value for a core field using a multilingual dict.
+    """
+    new_key = ()
+    if key[0] == 'resources':
+        new_key = key[0:2] + (key[2].replace('_lang', ''),)
+    else:
+        new_key = (key[0].replace('_lang', ''),) + key[1:]
```

Comment:
You need an `if k.endswith(LANG_SUFFIX):` in here to prevent overwriting values that don't have a name with the correct form, and a `k = k[:-len(LANG_SUFFIX)]` instead to not accidentally remove `"_lang"` strings inside the name